### ISSUE TITLE
Geckolib failed to load due to missing language provider

### ISSUE THREAD

**Issue by pvsmagbojos** [bernie-g/geckolib#263]
Mod Version
3.1.0

Mod Loader
Forge

Mod Loader Version
40.1.48

What happened?
Error on load: Mod File geckolib-forge-1.19-3.1.0.jar needs language provider javafml:41 or above to load
We have found 40

**Comment by pvsmagbojos** [comment]
It might be that this is intended for 1.19 version of Minecraft, though it was mistakenly tagged for 1.18.2 instead on the CurseForge website.

**Comment by breadguy69** [comment]
yeah I came here to report that the 1.19-3.1.0 jar is marked for 1.18.2 on cf
